### PR TITLE
feat: warning, alert, success buttons added

### PR DESCRIPTION
**Pull request by Yhuley**
https://www.figma.com/file/NAlqiKft9MuxX3QmGQCleQ/Valcori_Application?node-id=6573%3A92643
We need different status buttons: 'Warning', 'Alert', 'Success', in this UI-kit we only have Warning button but in future It can be useful to have different one.

### CODE REVIEW

**Comment by lazzystep** [comment]
Could you please provide more info about it. Ideally link to trello ticket as well. Thanks in advance!

File: src/components/button/button-DOCS.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Button component
 
 ```js
-import { PrimaryButton, SecondaryButton, Button, ButtonProps } from '@panenco/ui';
+import { PrimaryButton, SecondaryButton, Button, WarningButton, AlertButton, SucessButton, ButtonProps } from '@panenco/ui';
```
Comment (GolotaMishka) [inline src/components/button/button-DOCS.md:6]: 'Success' typo

**Review by GolotaMishka — CHANGES_REQUESTED** [review]

**Review by GolotaMishka — APPROVED** [review]

**Comment by lazzystep** [comment]
cc: @vlacher12

**Comment by vlacher12** [comment]
What problem does this PR solve? @Yhuley we can discuss in a separate meeting

**Comment by Yhuley** [comment]
> we can discuss in a separate meeting

Ok, maybe today?

**Review by sergdeev — APPROVED** [review]

**Pull request closed without merging**